Pipfile

```diff
@@ -31,6 +31,7 @@ requests = "~=2.27"
 slackclient = "~=2.5"
 sqlalchemy = "~=1.4"
 pymongo = "~=3.12.1"
+cryptography = "*"
```

Comment:
Do you know why this is required? Is it used by another library? If it is just needed locally i.e for testing or linting it can go in the dev-packages section